components/sensor/hlw8012.rst

```diff
@@ -55,8 +55,34 @@ Configuration variables:
   Defaults to the Sonoff POW's value ``2351``.
 - **change_mode_every** (*Optional*, int): After how many updates to cycle between the current/voltage measurement mode.
   Note that the first value after switching is discarded because it is often inaccurate. Defaults to ``8``.
+- **initial_mode** (*Optional*, string): The initial measurement mode. Defaults to ``VOLTAGE``.
```

Comment:
Create a new `Adanced Options:` block below update_interval and move this and `change_mode_every` there. (These options don't matter for ~3/4 of users). Example: servo.rst